src/components/MenuBar/MenuBarToggleButton.tsx

```
@@ -0,0 +1,50 @@
+import {
+    IconButton,
+    IconButtonProps,
+    Tooltip,
+    TooltipTypeMap,
+} from "@mui/joy";
+import { Check, Close } from "@mui/icons-material";
```

ESLint

[error] '@mui/icons-material' import is restricted from being used. Please use path imports and name your imports with postfix "Icon" to avoid confusions.

(no-restricted-imports)